monitoring/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.9'
+
+services:
+
+  app_python:
+    image: justmark0/devopsiu:latest
+    ports:
+      - 8000:8000
+    logging:
+      driver: "json-file"
+      options:
+        tag: "{{.ImageName}}|{{.Name}}"
```

Comment:
i don't get why you all need image naem but ok ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:) maybe im stupid